selftests/checkall

```diff
@@ -58,6 +60,23 @@ signed_off_check() {
     fi
 }
 
+
+results_dir_content() {
+    NOW="$(ls $RESULTS_DIR)"
+    if [ "$(echo $* | xargs)" != "$(echo $NOW | xargs)" ]; then
+        echo "The output of 'ls ~/avocado/job-results/' is not the same as before running the checks"
```

Comment:
Those mentions of `~/avocado/job-results/` should be changed to either "Avocado's configured job results directory" or the actual value.